### ISSUE TITLE
How to make folder and save image in dragdrop extension?

### ISSUE THREAD

**Issue by re4lfl0w** [ipython-contrib/jupyter_contrib_nbextensions#755]
I tried to drag and drop image in my jupyter notebook.
I check how the extension are saving image?

```
<img src='test.jpg'> in markdown. and save test.jpg in notebook same directory.
```

I want to save image in 'images' directory.
## Problem
1. How do I make images directory?
2. How to save image in images directory?

I saw main.js but I didn't understand.

**Comment by juhasch** [comment]
The path to the images is set as `IPython.notebook.path` in the `send_to_server()` function calls. You can add a subdirectory there. Ideally, this would be configurable and not hardcoded, like some other extensions already allow it.

To create a new subdirectory, I guess this would be possible using the contents API. You can look at the source code in the notebook at `notebook/static/services/contents.js` on how it is used.

**Comment by juhasch** [comment]
I had to fix the drag&drop extension, so I added this feature. Does this work for you ?

**Comment by jcb91** [comment]
assuming this was fixed by #758. Please reopen if not.
